### PR TITLE
Doc update & minor updates to test unit code

### PR DESCRIPTION
**Pull request by tgiphil**

### CODE REVIEW

File: Source/Mosa.Utility.UnitTests/Numbers/U1.cs [hunk 1]
```diff
@@ -31,8 +33,16 @@ public static IList<byte> GetSeries()
 		};
 
 		list = list.Distinct().ToList();
-		list.Sort();
 
+		//for (var i = 0; i < 8; i++)
```
Comment (charsleysa) [inline Source/Mosa.Utility.UnitTests/Numbers/U1.cs:37]: Leftover commented code, is this intentional?

File: Source/Mosa.Utility.UnitTests/Numbers/U2.cs [hunk 2]
```diff
@@ -35,6 +37,15 @@ public static IList<ushort> GetSeries()
 		};
 
 		list = list.Distinct().ToList();
+
+		//for (var i = 0; i < 16; i++)
```
Comment (charsleysa) [inline Source/Mosa.Utility.UnitTests/Numbers/U2.cs:41]: Leftover commented code, is this intentional?

File: Source/Mosa.Utility.UnitTests/Numbers/U4.cs [hunk 3]
```diff
@@ -39,6 +41,15 @@ public static IList<uint> GetSeries()
 		};
 
 		list = list.Distinct().ToList();
+
+		//for (var i = 0; i < 32; i++)
```
Comment (charsleysa) [inline Source/Mosa.Utility.UnitTests/Numbers/U4.cs:45]: Leftover commented code, is this intentional?

File: Source/Mosa.Utility.UnitTests/Numbers/U8.cs [hunk 4]
```diff
@@ -43,6 +45,15 @@ public static IList<ulong> GetSeries()
 		};
 
 		list = list.Distinct().ToList();
+
+		//for (var i = 0; i < 64; i++)
```
Comment (charsleysa) [inline Source/Mosa.Utility.UnitTests/Numbers/U8.cs:49]: Leftover commented code, is this intentional?

File: Source/Mosa.Utility.UnitTests/Numbers/I1.cs [hunk 5]
```diff
@@ -33,6 +35,15 @@ public static IList<sbyte> GetSeries()
 		AddNegatives(list);
 
 		list = list.Distinct().ToList();
+
+		//for (var i = 0; i < 8; i++)
```
[the remark this answers is not in the document]
Comment (tgiphil) [reply Source/Mosa.Utility.UnitTests/Numbers/I1.cs:39]: intentional - keep some code that blows up the number of unit tests

**Review by charsleysa — APPROVED** [review]
LGTM

**Pull request merged**